src/Google/Ads/GoogleAds/V1/Services/AccountBudgetProposalServiceClient.php

```diff
@@ -26,7 +26,7 @@
 
 namespace Google\Ads\GoogleAds\V1\Services;
 
-use Google\Ads\GoogleAds\Lib\V1\GoogleAdsGapicClientTrait;
+use Google\Ads\GoogleAds\Lib\GoogleAdsGapicClientTrait;
```

Comment:
This is wrong. Have you synced your workspace yet?

Reply:
Right, fixed this.